```
/var/cache/acbs/build/acbs.laggm7v3/fbthrift/thrift/lib/cpp2/gen/module_types_cpp.cpp:34:78: error: static assertion failed: oversized
   34 |     !FOLLY_F14_VECTOR_INTRINSICS_AVAILABLE || !folly::has_extended_alignment ||
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
   35 |         sizeof(enum_find<int>) <= folly::cacheline_align_v,
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/runtime-common/edencommon/spec
+++ b/runtime-common/edencommon/spec
@@ -1,4 +1,4 @@
-VER="2024.12.30.00"
+VER="2025.03.24.00"
 SRCS="git::commit=tags/v$VER::https://github.com/facebookexperimental/edencommon"
 CHKSUMS="SKIP"
 CHKUPDATE="anitya::id=375630"

--- a/runtime-network/fb303/autobuild/patches/0001-Search-FBTHRIFT_INCLUDE_DIR-when-generating-a-python.patch
+++ b/runtime-network/fb303/autobuild/patches/0001-Search-FBTHRIFT_INCLUDE_DIR-when-generating-a-python.patch
@@ -1,0 +1,25 @@
+From 174c3d3a914a7f6f158773afdca88c00f491bb73 Mon Sep 17 00:00:00 2001
+From: Henry Chen <henry.chen@oss.cipunited.com>
+Date: Thu, 20 Mar 2025 17:25:01 +0800
+Subject: [PATCH] Search ${FBTHRIFT_INCLUDE_DIR} when generating a python
+ library
+
+---
+ build/fbcode_builder/CMake/FBThriftPyLibrary.cmake | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/build/fbcode_builder/CMake/FBThriftPyLibrary.cmake b/build/fbcode_builder/CMake/FBThriftPyLibrary.cmake
+index fa77cde71..1aed8fbf4 100644
+--- a/build/fbcode_builder/CMake/FBThriftPyLibrary.cmake
++++ b/build/fbcode_builder/CMake/FBThriftPyLibrary.cmake
+@@ -87,6 +87,7 @@ function(add_fbthrift_py_library LIB_NAME THRIFT_FILE)
+       --legacy-strict
+       --gen "py:${GEN_ARG_STR}"
+       "${thrift_include_options}"
++      -I "${FBTHRIFT_INCLUDE_DIR}"
+       -o "${output_dir}"
+       "${CMAKE_CURRENT_SOURCE_DIR}/${THRIFT_FILE}"
+     WORKING_DIRECTORY
+-- 
+2.49.0
+

--- a/runtime-network/fb303/spec
+++ b/runtime-network/fb303/spec
@@ -1,4 +1,4 @@
-VER="2024.12.30.00"
+VER="2025.03.24.00"
 SRCS="git::commit=tags/v$VER::https://github.com/facebook/fb303.git"
 CHKSUMS="SKIP"
 CHKUPDATE="anitya::id=242787"

--- a/runtime-network/mvfst/spec
+++ b/runtime-network/mvfst/spec
@@ -1,4 +1,4 @@
-VER="2024.12.30.00"
+VER="2025.03.24.00"
 SRCS="git::commit=tags/v$VER::https://github.com/facebook/mvfst.git"
 CHKSUMS="SKIP"
 CHKUPDATE="anitya::id=373578"

